roles/sro_run_e2e-test/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Git checkout
+  ansible.builtin.git:
+    repo: "{{ sro_git_repo }}"
+    dest: /tmp/special-resource-operator
+    version: "{{ sro_git_ref }}"
+    force: yes
+
+      
```

Comment:
extra empty line here, 
and multiple newline-at-end-of-file missing (see github showing the red "forbidden way" circle